images/linux/scripts/installers/swig.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+################################################################################
+##  File:  swig.sh
+##  Desc:  Installs Swig
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+# Install Swig
+apt-get update -y
```

Comment:
We don't need do `update` command if we don't add new repo

Reply:
Fixed it